fabtests/test_configs/udp/udp.exclude

```diff
@@ -1,16 +1,19 @@
 # Regex patterns of tests to exclude in runfabtests.sh
 
+# No utility provider available for MSG EP support
 ^msg
 -e msg
-rdm
-poll
-av_test.*
-eq_test
-mr_test
 cm_data
+
+# Issues in ofi_rxd provider prevent tests from passing
```

Comment:
btw - rxd works fine with cntr_pingpong, cq_data, and poll
rdm_tagged_peek doesn't work, not because of issues in the provider per se but lacks support for it so it should probably go in the below section
IIRC, I included the tests that have polling/waiting in the exclude file because of Travis testing on OSX since it doesn't have epoll. Those should probably go in the osx.exclude file instead.

Reply:
(and I should add that the cntr_pingpong test works fine *after* the PR I added fixing the rxd counters)

Reply:
Removed the cntr, cq_data, and poll tests from the exclude, moved tagged down.